SwiftGit2/Objects.swift

```diff
@@ -222,3 +222,60 @@ public struct Tag: ObjectType, Hashable {
 		message = String(validatingUTF8: git_tag_message(pointer))!
 	}
 }
+
+public struct StatusOptions: OptionSet {
```

Comment:
@spekke it's good to create a new file for this `StatusOptions` just like `CheckoutStrategy`

Reply:
I agree 👍